storage/src/rocksdb_client.cpp

```diff
@@ -293,6 +299,7 @@ Status Client::multiGet(const KeysVector &_keysVec, OUT ValuesVector &_valuesVec
     }
     _valuesVec.push_back(Sliver(std::move(values[i])));
   }
+  tryToUpdateMetrics();
```

Comment:
Will this call update with the correct count of keys fetched (i.e. possibly more than 1)?

Reply:
Yeah. It takes the internal rocksdb metrics